docs/guide/en/structure/entry-script.md

```diff
@@ -103,5 +95,5 @@ try {
 
 For alternative runtimes such as RoadRunner or Swoole, special entry scripts should be used. See:
 
-- [Using Yii with RoadRunner](../tutorial/using-yii-with-roadrunner.md)
-- [Using Yii with Swoole](../tutorial/using-yii-with-swoole.md)
+* [Using Yii with RoadRunner](../tutorial/using-yii-with-roadrunner.md)
+* [Using Yii with Swoole](../tutorial/using-yii-with-swoole.md)
```

Comment:
It was OK as is.

Reply:
this what? all?

Reply:
List items with `-` instead of `*`.

Reply:
I have found many occurrences of '*'. VSCode reports error and cites (https://github.com/DavidAnson/markdownlint/blob/v0.34.0/doc/md004.md#md004---unordered-list-style).
What should I do, as I find '*' along the way?

"List items with - instead of *." for every occurrence I encounter?
(I will have to do it manually one by one so as not to affect code snippets)

Reply:
Yes, `-`.